modules/cpdos/hbh.py

```diff
@@ -10,28 +10,21 @@
 
 VULN_NAME = "Hop-By-Hop"
 
-# Function to make requests and compare responses
-def compareRequests(url, s, headers, resp1, params2, authent):
-    try:
-        resp2 = s.get(url, headers=headers, params=params2, auth=authent, allow_redirects=False, verify=False, timeout=10)
-        return resp2
-    except requests.exceptions.ConnectionError as e:
-        #print(f"Error : {e}")
-        return None
-
-
```

Comment:
Removed the function as there were no more logic besides the passthrough to get request